src/operation/model_numerical_analysis_utils.jl

```diff
@@ -106,10 +106,19 @@ function get_constraint_numerical_bounds(model::OperationModel)
     end
     bounds = ConstraintBounds()
     for (const_key, constriant_array) in get_constraints(get_optimization_container(model))
-        for idx in Iterators.product(constriant_array.axes...)
-            con_obj = JuMP.constraint_object(constriant_array[idx...])
-            update_coefficient_bounds(bounds, con_obj, (const_key, idx))
-            update_rhs_bounds(bounds, con_obj, (const_key, idx))
+        if typeof(constriant_array) <: JuMP.Containers.SparseAxisArray
```

Comment:
1. Add a TODO to handle this at compile and not at run time
2. 1
```suggestion
        if isa(constriant_array, JuMP.Containers.SparseAxisArray)
```